submissions/Lu-sure/Popup/index.html

```diff
@@ -0,0 +1,168 @@
+                      Mail
+                    </a>
+                  </li>
+                  <li>
+                    <a href="#" class="popup-item">
+                      <img src="popup-icons/news.png" alt="News icon" class="popup-item-img">
+                      News
+                    </a>
+                  </li>
+                  <li>
+                    <a href="#" class="popup-item">
+                      <img src="popup-icons/maps.png" alt="Map icon" class="popup-item-img">
+                      Map
+                    </a>
+                  </li>
+                  <li>
+                    <a href="#" class="popup-item">
+                      <img src="popup-icons/files.png" alt="Files icon" class="popup-item-img">
+                      Files
+                    </a>
+                  </li>
+                  <li>
+                    <a href="#" class="popup-item">
+                      <img src="popup-icons/contacts.png" alt="Contacts icon" class="popup-item-img">
+                      Contacts
+                    </a>
+                  </li>
+                  <li>
+                    <a href="#" class="popup-item">
+                      <img src="popup-icons/tanslation.png" alt="Translater icon" class="popup-item-img">
+                      Translate
+                    </a>
+                  </li>
+                  <li>
+                    <a href="#" class="popup-item">
+                      <img src="popup-icons/videos.png" alt="Videos icon" class="popup-item-img">
+                      Videos
+                    </a>
+                  </li>
+                  <li>
+                    <a href="#" class="popup-item">
+                      <img src="popup-icons/search.png" alt="Search icon" class="popup-item-img">
+                      Search
+                    </a>
+                  </li>
+                  <li>
+                    <a href="#" class="popup-item">
+                      <img src="popup-icons/photos.png" alt="Photo icon" class="popup-item-img">
+                      Photo
+                    </a>
+                  </li>
+                  <li>
+                    <a href="#" class="popup-item">
+                      <img src="popup-icons/apps.png" alt="Apps icon" class="popup-item-img">
+                      Apps
+                    </a>
+                  </li>
+                  <li>
+                    <a href="#" class="popup-item">
+                      <img src="popup-icons/documents.png" alt="Documents icon" class="popup-item-img">
+                      Documents
+                    </a>
+                  </li>
+                </ul>
+                
+                <input type="checkbox" id="more-button" class="hide-checkbox more" tabindex="1">
+                <label for="more-button" class="more-bt-label" tabindex="0"><span class="more-bt-text">More</span></label>
+
+                <hr class="popup-divider">
```

Comment:
Avoid adding decorative elements into HTML markup, use CSS for it.

Reply:
Can you help me, how to do this? Point me what to read, please))

Reply:
It's just a line, You can create it by `border-top` or `border-bottom` or using pseudoelements `::before`, `::after`. And any of these ways will be created by CSS and without unnecessary HTML elements in markup.).
Also, it has an influence on accessibility. [Read more](https://developer.mozilla.org/en-US/docs/Learn/Accessibility/HTML).

Reply:
I understand, it was quite not efficient. And your variants are so obvious! Thank you!

Reply:
Not fixed yet.

Reply:
Fixed already.